src/common/styles/flex.css

```diff
@@ -1,3 +1,15 @@
 .displayFlex{
     display: flex;
+}
+.flexColumn{
+    flex-direction: column;
+}
+.flexWrap{
+    flex-wrap: wrap;
+}
+.flexAlignItemsCenter{
+    align-items: center;
+}
+.flexJustifyContentCenter{
+    justify-content: center;
```

Comment:
Added some additional global flex css. 
I want to try and use flex when we can and also add common flex types into this file.